app/config/settings.py

```diff
@@ -1089,6 +1089,21 @@ def strtobool(val) -> bool:
 CODEBUILD_ACCESS_KEY = os.environ.get("CODEBUILD_ACCESS_KEY", "")
 CODEBUILD_SECRET_KEY = os.environ.get("CODEBUILD_SECRET_KEY", "")
 CODEBUILD_REGION = os.environ.get("CODEBUILD_REGION", "")
+CODEBUILD_ONLY_ALLOW_VERIFIED = strtobool(
```

Comment:
This setting can be removed, we only want verified users to do this. Maybe add the verified checks to the views too?